src/main/resources/application-h2dev.properties

```diff
@@ -0,0 +1,15 @@
+spring.h2.console.path=/review
```

Comment:
should remove unused configuration and have in groups and alphabetical order

Reply:
regrouped, but some order makes more sense together as opposed to alphabetical. for example, username and password.